radicle-cli/src/commands/checkout.rs

```diff
@@ -19,25 +18,23 @@ pub const HELP: Help = Help {
     usage: r#"
 Usage
 
-    rad checkout <id> [<option>...]
+    rad checkout --remote <did> [<option>...]
```

Comment:
It should be `rad checkout <rid> [--remote <did>]`

Reply:
https://github.com/radicle-dev/heartwood/pull/445/commits/6d0b338c6920de263d264d7bac5e5c819bc80511#diff-8a005577f622e3c2fbacc399071a2229c125a669390adec38936afe6ca498b36R21